sound/soc/sof/core.c

```diff
@@ -339,6 +341,14 @@ int snd_sof_device_probe(struct device *dev, struct snd_sof_pdata *plat_data)
 }
 EXPORT_SYMBOL(snd_sof_device_probe);
 
+bool snd_sof_device_probe_completed(struct device *dev)
+{
+	struct snd_sof_dev *sdev = dev_get_drvdata(dev);
+	return sdev->probe_completed;
+}
+
```

Comment:
usually there's no empty line here. To the commit message:
> Current delayed probe implementation does not call sof_probe_complete,

you meant in an error case, right?

Reply:
Ack @lyakh on both, will address in next version. Thanks!